cmd/get.go

```diff
@@ -13,6 +13,32 @@ import (
 	"github.com/spf13/cobra"
 )
 
+type outputEnum string
+
+const (
+	outputStandard outputEnum = "standard"
+	outputNotes    outputEnum = "notes"
+)
+
+func (e *outputEnum) String() string {
+	return string(*e)
+}
+
+func (e *outputEnum) Set(v string) error {
+	switch v {
+	case string(outputStandard), string(outputNotes):
+		*e = outputEnum(v)
+		return nil
+	default:
+		return fmt.Errorf(`must be one of %s or %s`, outputStandard, outputNotes)
+	}
+}
+
+func (e *outputEnum) Type() string {
+	return "outputEnum"
+}
+
+var outputTemplate = outputStandard
```

Comment:
`outputTemplate` could become `outputType` which is just a string.

```suggestion
var outputType string
```